Backend/app/rider-platform/rider-app/Main/src/Storage/Tabular/Maps.hs

```diff
@@ -0,0 +1,65 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Storage.Tabular.Maps where
```

Comment:
usually module name correspond to sql table name, like:

```
Storage.Tabular.PlaceNameCache
Domain.Types..PlaceNameCache
```
and so on
If you consider that it would be other entities related with maps, then you can create separate folder for this:

```
Storage.Tabular.Maps.PlaceNameCache
Domain.Types..Maps.PlaceNameCache
```

Reply:
ok